Source/WebCore/page/EventHandler.h

```diff
@@ -655,6 +655,7 @@ class EventHandler {
 
     TouchArray m_touches;
     RefPtr<LocalFrame> m_touchEventTargetSubframe;
+    HashMap<unsigned, std::pair<IntPoint, IntPoint>, WTF::IntHash<unsigned>, WTF::UnsignedWithZeroKeyHashTraits<unsigned>> m_touchLastGlobalPositionAndDeltaMap;
```

Comment:
are the default HashMap template arguments not sufficient?

Reply:
No, we need this because there can be touch IDs (map key) with value `0`, and that corrupts the hash table.

Reply:
Well, we really only need `WTF::UnsignedWithZeroKeyHashTraits<unsigned>`, but the `IntHash<>` template arg gets in the way 😛